README.md

```diff
@@ -137,7 +137,7 @@ If you're not fully familiar with form validation in Angular, these may be helpf
 * [Angular Form Validation (Scotch.io)](http://scotch.io/tutorials/javascript/angularjs-form-validation)
 * [Form validation with AngularJS (ng-newsletter)](http://www.ng-newsletter.com/posts/validations.html)
 
-angular-credit-cards sets validity keys that match the directive names (`ccNumber`, `ccCvc`, `ccExp`, `ccExpMonth`, `ccExpYear`). You can use these keys or the form css classes in order to display error messages.
+angular-credit-cards sets validity keys that match the directive names (`ccNumber`, `ccCvc`, `ccExp`, `ccExpMonth`, `ccExpYear`). You can use these keys or the form css classes in order to display error messages. If input is unparseable (letters, empty string), a `parse` key is set per Angular conventions.
```

Comment:
Can you swap this out to reflect that this is a framework feature and not adherence to a convention?

> a `parse` key is set per Angular conventions.

"Angular sets a `parse` key instead."